src/components/MapDisplay.css

```diff
@@ -0,0 +1,4 @@
+#map {
+    width: 500px;
+    height: 400px;
```

Comment:
can make use of pf variables

Reply:
Currently not feasible since we do not use SASS.